src/get-started/index.md.njk

```diff
@@ -7,10 +7,21 @@ show_subnav: true
 
 The examples in the GOV.UK Design System come with code to make it easy for you to use them in your project. 
 
-Get started:
-
+There are guides to getting started:
 
 - [prototyping](prototyping)
 - [in production](production)
 
-If you’ve used GOV.UK Elements, GOV.UK Template or the GOV.UK Frontend Toolkit before, you might also find it useful to read the guide on [updating your code](updating-your-code).
+You may find the [guide on updating your code](updating-your-code) useful if you’re currently working with one of our legacy products - GOV.UK Elements, GOV.UK Template or the GOV.UK Frontend Toolkit.
+
+## Using styles, components and patterns
+
+When something is published in the design system as a [style](/styles/), [component](/components/) or [pattern](/patterns/) we include details of how and when it’s been tested in user research. This should help you decide whether it’s something you can re-use or adapt for your service.
+
+You can ask questions or share your views on how to use a style, component or pattern by joining the discussion on GitHub. There are links at the end of each style, component and pattern page - under the ‘Help improve this page’ heading.
+
+## GitHub discussions about styles, components and patterns
+
+Bear in mind that ideas discussed on GitHub may not have been tested. You can use them as a starting point, but it’s important to carry out user research to check that they work for your service. Then when you’ve carried out your user research, add your findings to the relevant discussion.
```

Comment:
I thought it best to talk about GitHub in general because people are likely to access the discussion via the link at the end of the relevant style / pattern / component page - so they won't necessarily have a mental model of the backlog as a thing.

Does that make sense?